lorawan-encoding/src/maccommandcreator.rs

```diff
@@ -612,11 +660,13 @@ impl RXTimingSetupReqCreator {
 /// let creator = lorawan::maccommandcreator::RXTimingSetupAnsCreator::new();
 /// let res = creator.build();
 /// ```
+#[derive(Debug)]
+#[cfg_attr(feature = "defmt", derive(defmt::Format))]
 pub struct RXTimingSetupAnsCreator {}
 
 impl_mac_cmd_creator_boilerplate!(RXTimingSetupAnsCreator, 0x08);
 
-#[derive(Debug, PartialEq, Eq)]
```

Comment:
Is there reasoning in removing PartialEq, Eq?

Reply:
my reasoning was that they were implemented only for some commands and not all. I also don't see a reason for comparing two maccommandcreator objects. I can add them back if needed, Ill try to check in git blame why they are there in the first place.

Reply:
Turns out I am to blame. I added them half-assedly when I first added the 1.0.4 commands.